docs/documentation/faq.md

```diff
@@ -19,6 +19,40 @@ make a nice name for an Eclipse plug-in about Pair Programming, where
 driver and observer cycle their roles while revolving around code of
 interest.
 
+### How does Saros compare to other collaborative real-time editing tools?
+
+These tables compare known alternatives to Saros.
+
+In the following we are using these definitions:
+* **Open-Source** - A software using a license contained in the [list of approved licenses of the Open Source Initiative](https://opensource.org/licenses/alphabetical)
+* **Commercial** - It is not possible to use the software for free without limitations.
+* **Freeware** - You can use the software for free, but the code is not published.
+* **Self-hosted** - The possibility to host a corresponding server that manages the connection or also the shared workspaces.
+
+#### Web-based IDEs
```

Comment:
How about replacing `None` with `No` for self-hosted. (None sounds weird as the opposite to `Yes`.)

You could also replace the option `costly` with something like `subscription required` or `(yes)*` and annotating it with `*Subscription required` or `*Premium feature`. Like this:

---
|Name         |IDE                                              |Category    |Self-hosted |
|-------------|-------------------------------------------------|------------|------------|
|FlooBits     |Atom, Emacs, IntelliJ IDEA, Neovim, Sublime Text |Commercial  |(Yes)*      |
|Saros        |Eclipse, IntelliJ IDEA                           |Open-Source |Yes         |
|Live Share   |Visual Studio (Code)                             |Freeware    |No        |
|Teletype     |Atom                                             |Open-Source |Yes         |

_*Subscription required_

---

Reply:
Substituted "None" with "No" and the entry "Costly" with "Yes", because the fact that the whole software is not for free is already implied by "Commercial". It should be evident that the self-hosted version is also not for free.